### DISCUSSION TITLE
Deadlock for a time invariant in UPPAAL 4.1.25-5 but not in UPPAAL 4.0.15

### DISCUSSION THREAD

**Discussion by AlCourbis** [UPPAALModelChecker/UPPAAL-Meta#125] [Show and tell]
I have defined a simple process with a time invariant x<=5 at the initial location and an exit transition with guard : x==5 going out to a location.
UPPAAL 4.1.25-5 detects a deadlock for x> 5. However, in my previous release, UPPAAL 4.0.15, no deadlock is detected.
I do not understand the verdict of UPPAAL 4.1.25-5 ! What is wrong  ?
I would appreciate some explanations...
Many thanks !

**Comment by mikucionisaau** [comment]
I presume the deadlock you see is displayed in the symbolic simulator?
Those are spurious, i.e. the system does not exist beyond invariant `x<=5` thus can be safely ignored. This issue is fixed in the upcoming release.
If you see deadlocks in verifier, then I'd like to know more.
